src/test/regress/expected/propagate_extension_commands.out

```diff
@@ -0,0 +1,399 @@
+              30
+(1 row)
+
+-- show that the extension is created on both existing and new node
+SELECT run_command_on_workers($$SELECT count(extnamespace) FROM pg_extension WHERE extname = 'seg'$$);
+ run_command_on_workers 
+------------------------
+ (localhost,57637,t,1)
+ (localhost,57638,t,1)
+(2 rows)
+
+SELECT run_command_on_workers($$SELECT extversion FROM pg_extension WHERE extname = 'seg'$$);
+ run_command_on_workers  
+-------------------------
+ (localhost,57637,t,1.3)
+ (localhost,57638,t,1.3)
+(2 rows)
+
+-- and similarly check for the reference table
+select count(*) from pg_dist_partition where partmethod='n' and logicalrelid='ref_table_2'::regclass; 
+ count 
+-------
+     1
+(1 row)
+
+SELECT count(*) FROM pg_dist_shard WHERE logicalrelid='ref_table_2'::regclass;
+ count 
+-------
+     1
+(1 row)
+
+DROP TABLE ref_table_2;
+-- now test create extension in another transaction block but rollback this time
+BEGIN;
+	CREATE EXTENSION isn WITH VERSION '1.1' SCHEMA public;
+ROLLBACK;
+-- at the end of the transaction block, we did not create isn extension in coordinator or worker nodes as we rollback'ed
+-- make sure that the extension is not distributed
+SELECT count(*) FROM citus.pg_dist_object WHERE objid = (SELECT oid FROM pg_extension WHERE extname = 'isn');
+ count 
+-------
+     0
+(1 row)
+
+-- and the extension does not exist on workers
+SELECT run_command_on_workers($$SELECT count(*) FROM pg_extension WHERE extname = 'isn'$$);
+ run_command_on_workers 
+------------------------
+ (localhost,57637,t,0)
+ (localhost,57638,t,0)
+(2 rows)
+
+-- give a notice for the following commands saying that it is not
+-- propagated to the workers. the user should run it manually on the workers 
+CREATE TABLE t1 (A int);
+CREATE VIEW v1 AS select * from t1;
+ALTER EXTENSION seg ADD VIEW v1;
+NOTICE:  Citus does not propagate adding/dropping member objects
+HINT:  You can add/drop the member objects on the workers as well.
+ALTER EXTENSION seg DROP VIEW v1;
+NOTICE:  Citus does not propagate adding/dropping member objects
+HINT:  You can add/drop the member objects on the workers as well.
+DROP VIEW v1;
+DROP TABLE t1;
+-- drop multiple extensions at the same time
+CREATE EXTENSION isn WITH VERSION '1.1' SCHEMA public;
+-- let's create another extension locally
+set citus.enable_ddl_propagation to 'off';
+CREATE EXTENSION pg_buffercache;
+set citus.enable_ddl_propagation to 'on';
+-- SET client_min_messages TO WARNING before executing a DROP EXTENSION statement
+SET client_min_messages TO WARNING;
+DROP EXTENSION pg_buffercache, isn CASCADE;
+SELECT count(*) FROM pg_extension WHERE extname IN ('pg_buffercache', 'isn');
+ count 
+-------
+     0
+(1 row)
+
+-- restore client_min_messages after DROP EXTENSION
+RESET client_min_messages;
+-- SET client_min_messages TO WARNING before executing a DROP EXTENSION statement
+SET client_min_messages TO WARNING;
+-- drop extension in a transaction block, should just work
+--BEGIN;
```

Comment:
undo the comments for `BEGIN/COMMIT`, and update the comment below -- TODO:: There is a bug now